pkg/ibmvpc/examples/acl_testing5subnetsDiff.txt

```diff
@@ -1,9 +1,9 @@
 Analysis for diff between VPC test-vpc-ky1 and VPC test-vpc-ky2
--- sub1-2-ky => sub1-1-ky : missing connection 
--- sub1-3-ky => sub1-1-ky : missing connection 
+diff-type: added source: sub1-2-ky destination: sub1-1-ky dir1: protocol: TCP dir2: No connection
```

Comment:
OK

Reply:
> so an item indicating if one of the subnets was added/removed, appears only if relevant? would it help to add this item with empty value in case the subnet was not added/removed, for consistency of lines?

You mean write "subnets-diff-info:" and keep it empty?

Reply:
Yes, I think adding this part `subnets-diff-info` will let the output lines be more consistent, and also match the other planned formats (e.g. `md` format as a table where this is another column that either has an empty cell or contain value..).

Reply:
done, though I don't like the way it looks now